public/app/features/alerting/unified/components/silences/SilencesTable.tsx

```diff
@@ -29,12 +33,35 @@ export interface SilenceTableItem extends Silence {
 type SilenceTableColumnProps = DynamicTableColumnProps<SilenceTableItem>;
 type SilenceTableItemProps = DynamicTableItemProps<SilenceTableItem>;
 interface Props {
-  silences: Silence[];
-  alertManagerAlerts: AlertmanagerAlert[];
   alertManagerSourceName: string;
 }
 
-const SilencesTable = ({ silences, alertManagerAlerts, alertManagerSourceName }: Props) => {
+const SilencesTable = ({ alertManagerSourceName }: Props) => {
+  const { data: alertManagerAlerts = [], isLoading: amAlertsIsLoading } =
+    alertmanagerApi.endpoints.getAlertmanagerAlerts.useQuery(
+      { amSourceName: alertManagerSourceName, filter: { silenced: true, active: true, inhibited: true } },
+      { pollingInterval: SILENCES_POLL_INTERVAL_MS }
```

Comment:
nit: perhaps we should also add `refetchOnFocus` for our RTKQ hooks here? 20s is pretty slow and the API responses should be quite fast since the silences are kept in-memory in Alertmanager.